src/Nerdbank.Algorithms/Nerdbank.Algorithms.csproj

```diff
@@ -1,12 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
.NET 7 falls out of support next month. It hardly seems worthwhile to add support for it explicitly now. 
And anyway, it and net8.0 adds nothing that the pre-existing net6.0 target already offers. So let's revert the whole change.

Reply:
The issue which could arise is if, the condition below has the net6.0 removed then we could potentially be forcing a downgrade of System.Collections.Immutable to v6 and then some people would view that as a breaking change. If you like me don't see that as a breaking change, then i can drop net 7 & 8, the sdk upgrade and adjust the condition

Reply:
Oh, you're saying that the netstandard2.0 build will want System.Collections.Immutable 7.0, but the net6.0 build will want the 6.0 version of that assembly? That's very subtle, but interesting now that you've pointed it out. I suppose the right fix for that would be to change to never ask for more than 6.0 of the package, even for the netstandard2.0. So can we proceed with my request, but also change Directory.Packages.props to reduce the version referenced to 6.0.0?

Reply:
Sure will drop dependency to v6 & run with net 6 as the TFM not net 7